js/about/preferences.js

```diff
@@ -311,6 +315,26 @@ class BitcoinDashboard extends ImmutableComponent {
       </div>
     }
   }
+  get exchangePanel () {
+    const url = this.props.ledgerData.getIn(['exchangeInfo', 'exchangeURL'])
+    const name = this.props.ledgerData.getIn(['exchangeInfo', 'exchangeName'])
+    // Call coinbasePanel if we don't have the URL or Name
+    if (!url || !name) {
+      return this.coinbasePanel
+    } else {
+      return <div className='panel'>
+        <div className='settingsPanelDivider'>
+          <span className='fa fa-credit-card' />
+          <div className='settingsListTitle' data-l10n-id='outsideUSAPayment' />
+        </div>
+        <div className='settingsPanelDivider'>
+          <a target='_blank' className='browserButton primaryButton' href={url}>
+            {name}
```

Comment:
Nice catch! I totally forgot to add the word 'visit' before the button like @bradleyrichter has in his designs! Fixed.